src/components/NavBar.tsx

```diff
@@ -40,9 +41,25 @@ function NavBar() {
 
   const renderedNavItems = navTitles.map((item) => renderNavItem(item))
 
+  const [prevScrollPos, setPrevScrollPos] = useState(0)
+  const [visible, setVisible] = useState(true)
```

Comment:
maybe isVisible would be better since it's a boolean

Reply:
resolved in 00cc5bc